src/file.c

```diff
@@ -57,10 +60,7 @@ void display_file(struct entry_t *entry, struct search_t *search, int y, int is_
 
     attron(A_BOLD);
 
-    if (strcmp(search->directory, "./") == 0)
-        remove_double(entry->data + 3, '/', filtered_line);
-    else
-        remove_double(entry->data , '/', filtered_line);
+    remove_double(entry->data , '/', filtered_line);
```

Comment:
I think we did that to remove "./" from the file name displayed above the results entries

Reply:
You're right I disabled it because of issues with the external parsers. I need to re-enable it for the native parser.

Reply:
Done!
I also moved the path formatting into the `create_file` function.